orchestration/pom.xml

```diff
@@ -140,72 +144,37 @@
           <plugin>
-            <groupId>org.openapitools</groupId>
+            <groupId>com.sap.cloud.sdk.datamodel</groupId>
             <artifactId>openapi-generator-maven-plugin</artifactId>
+            <configuration>
+              <outputDirectory>${project.basedir}/src/main/java</outputDirectory>
+              <apiMaturity>released</apiMaturity>
+              <enableOneOfAnyOfGeneration>true</enableOneOfAnyOfGeneration>
+              <compileScope>COMPILE</compileScope>
+              <deleteOutputDirectory>true</deleteOutputDirectory>
+            </configuration>
             <executions>
               <execution>
+                <id>orchestration</id>
                 <goals>
                   <goal>generate</goal>
                 </goals>
                 <phase>generate-sources</phase>
                 <configuration>
-                  <!-- Specify the input OpenAPI spec file -->
                   <inputSpec>${project.basedir}/src/main/resources/spec/orchestration.yaml</inputSpec>
-                  <output>${project.basedir}</output>
-                  <!-- Specify the generator to use, e.g., java, spring, kotlin, etc. -->
-                  <generatorName>java</generatorName>
-                  <!-- Specify the package names for models, APIs, and invokers -->
+                  <apiPackage>com.sap.ai.sdk.orchestration.client</apiPackage>
                   <modelPackage>com.sap.ai.sdk.orchestration.client.model</modelPackage>
-                  <apiPackage>com.sap.ai.sdk.orchestration.client.api</apiPackage>
-                  <invokerPackage>com.sap.ai.sdk.orchestration.client.invoker</invokerPackage>
-
-                  <!-- Global properties level; can be unpacked with 'generate' prefix-->
-                  <globalProperties>
-                    <apiDocs>false</apiDocs>
-                    <modelDocs>false</modelDocs>
-                    <modelTests>false</modelTests>
-                    <apiTests>false</apiTests>
-                    <minimalUpdate>true</minimalUpdate>
-                  </globalProperties>
-
-                  <!-- Generator Specific properties level; some can be unpacked-->
-                  <configOptions>
-                    <generateBuilders>true</generateBuilders>
-                    <failOnUnknownProperties>false</failOnUnknownProperties>
-                    <hideGenerationTimestamp>true</hideGenerationTimestamp>
-                    <disallowAdditionalPropertiesIfNotPresent>false</disallowAdditionalPropertiesIfNotPresent>
+                  <additionalProperties>
+                    <pojoBuilderMethodName>create</pojoBuilderMethodName>
+                    <pojoBuildMethodName/>
+                    <pojoConstructorVisibility>protected</pojoConstructorVisibility>
                     <enumUnknownDefaultCase>true</enumUnknownDefaultCase>
-                    <useBeanValidation>false</useBeanValidation>
                     <useOneOfInterfaces>true</useOneOfInterfaces>
```

Comment:
`useOneOfInterfaces` is the only addition compared to the old Cloud SDK generator